src/Components/NavCategoryBar/NavCategoryBar.component.js

```diff
@@ -9,6 +9,14 @@ import { CART } from "../../config";
 import "./NavCategoryBar.styles.scss";
 
 class NavCategoryBar extends Component {
+  checkCartItemPageAccess = () => {
+    const { userToken } = this.props;
+    if (!userToken) {
+      alert("로그인을 해 주세요!");
+      this.props.history.push("/login");
+    } else this.props.history.push("/cartItems");
+  };
```

Comment:
```suggestion
  checkCartItemPageAccess = () => {
    const { userToken } = this.props;
    if (!userToken) {
      alert("로그인을 해 주세요!");
      return this.props.history.push("/login");
    }
    
    this.props.history.push("/cartItems");
  };
```
early return 애용해주세요~

Reply:
네 알겠습니다!